moabb/paradigms/motor_imagery.py

```diff
@@ -82,43 +99,90 @@ def scoring(self):
         return 'accuracy'
 
 
-class ImageryNClass(BaseMotorImagery):
-    """Imagery for multi class classification
+class MotorImageryMultiPass(BaseMotorImagery):
```

Comment:
this could actually be the main class, since `SinglePass` is the same code with just one single frequency band ?

Reply:
True, do you find it is more intuitive that way? I don't think it changes the actual structure of the code at all

Reply:
i don't have a strong opinion. i just saw a bunch of filtering function duplicated. 
i'm fine either way.

Reply:
Then I'll keep it how it is for now